README.md

```diff
@@ -1,43 +1,53 @@
-# Hello-ProtoData
-An example on code generation with ProtoData.
+[![Build Status][ubuntu-build-badge]][gh-actions]
+[![Build Status][windows-build-badge]][gh-actions]
+[![license][license-badge]](http://www.apache.org/licenses/LICENSE-2.0)
 
-This example demonstrates how to define a new Protobuf option and
-generate the validation code in response to this option
-using ProtoData API.
+[gh-actions]: https://github.com/spine-examples/Hello-ProtoData/actions
+[ubuntu-build-badge]: https://github.com/spine-examples/Hello-ProtoData/actions/workflows/build-on-ubuntu.yml/badge.svg
+[windows-build-badge]: https://github.com/spine-examples/Hello-ProtoData/actions/workflows/build-on-windows.yml/badge.svg
+[license-badge]: https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat
 
-### Custom Protobuf Option
+# Introduction
 
-Below is example on Protobuf message definition 
-with custom `size` option that is used for validating 
-the size of a repeated field.
+Declarations in a `.proto` file can be annotated with 
+a number of [options](https://protobuf.dev/programming-guides/proto3/#options). 
+Options do not change the overall meaning of a declaration, 
+but may affect the way it is handled in a particular context.
+
+Protobuf supports different types of options, e.g. file-level options, 
+message-level options, field-level options, etc. The complete list of 
```

Comment:
This is not a list of options. This is rather a list of available option types.

As anyone is able to come up with their own options, it is technically impossible to provide a link to a "complete list of available options".